src/com/googlecode/jmxtrans/model/naming/KeyUtils.java

```diff
@@ -254,7 +254,10 @@ private static String getTypeNameValuesSeparator(Query query) {
 		String[] result;
 		String[] keys = typeNameToken.split("=", 2);
 
-		if (keys.length == 2) {
+		for (int i=0; i<keys.length; i++) {
+		}
```

Comment:
Yeah, do not commit from Starbucks. Go to a place where thy have good coffee ... ;-)